### ISSUE TITLE
Objects fall through a simple collada plane/floor

### ISSUE THREAD

**Issue by 7ammer** [donmccurdy/aframe-physics-system#9]
I've noticed an issue where objects are falling through a simple Collada based floor/plane.

Here is the jsfiddle example:
https://jsfiddle.net/9z9e1jtk/

**Comment by donmccurdy** [comment]
Hi @7ammer — there's some more detail about this in https://github.com/donmccurdy/aframe-extras/issues/41. 

**tl;dr** — By default, your Collada model is turned into a CANNON.Trimesh shape. These are only partially supported by Cannon.js, have poor performance, and can only collide with Sphere and Plane shapes, as described in [Cannon.js collision matrix](https://github.com/schteppe/cannon.js#features).

In general cases, these are the options:

1. Use a single `static-body="shape:box"` or `static-body="shape:hull"` shape, if your terrain can be treated as convex. [Shape docs](https://github.com/donmccurdy/aframe-physics-system#body-shapes).
2. Use some combination of invisible boxes and planes to model the terrain.
3. Construct a shape programmatically using Cannon.js JavaScript APIs and add it to the A-Frame physics system.
4. Use a CANNON.Heightfield shape. Doing this automatically is not yet implemented, and I'm tracking the feature in https://github.com/donmccurdy/three-to-cannon/issues/1.

In your particular case, since this is just a very basic plane, I would normally use option (1).

```html
<a-entity static-body="shape: box" collada-model="#ground"></a-entity>
```

Unfortunately there is a bug with fitting a box to an infinitely thin object, so that won't work right now. I'll try to fix that. In the meantime, you'll need to use option (2) instead:

```html
<a-entity collada-model="#ground"></a-entity>
<a-plane static-body visible="false" height="100" width="100" rotation="-90 0 0"></a-plane>
```

**Comment by 7ammer** [comment]
Ah, I see! Thanks for letting me know.

Is .obj support in the same boat as .dae or is the support slightly better?

By the way, I tried `static-body="shape:hull"` on a collada object and it froze my browser. The same object written as an .obj file worked fine.

**Comment by donmccurdy** [comment]
I'd have expected OBJ or DAE to behave the same way, it should just depend on the geometry. I'm not sure why the `hull` option would have frozen, though.

**Comment by donmccurdy** [comment]
Bug related to infinitely-thin objects is now filed separately here: https://github.com/donmccurdy/aframe-physics-system/issues/45

I'm going to try to put together some info on how to use model(s) as your base scene for physics, soon, but that will require some preparation of the model in Blender... will never be as simple as dropping in an arbitrary model and putting `static-body` on it unfortunately.

Closing this issue in the meantime.
